src/main/java/se/sundsvall/document/api/model/DocumentDataUpdateRequest.java

```diff
@@ -0,0 +1,68 @@
+package se.sundsvall.document.api.model;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+import java.util.Objects;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+
+@Schema(description = "DocumentDataUpdateRequest model.")
+public class DocumentDataUpdateRequest {
+
+	@NotBlank
+	@Schema(description = "Actor that created this object", example = "username123", requiredMode = REQUIRED)
+	private String createdBy;
```

Comment:
Should createdBy be editable?

Reply:
The attribute are called "createdBy", since we don't work with changes/modifications (objects are treated as immutable)
I.e. all "modifications" will result in a new object (revision) and the value in the cratedBy will be stored in the corrsponding attribute on the new revision.

(We can of course describe this better in the "description".)

Reply:
Pushed enhanced (I hope) descriptions for these attributes